docs/earthfile/earthfile.md

```diff
@@ -304,6 +304,34 @@ final-target:
     COPY +intermediate/some-file.txt ./
 ```
 
+#### Examples
+
+Assuming the following directory tree, of a folder named `test`:
+
+```
+test
+  └── file
+```
+
+Here is how the following copy commands will behave:
+
+```
+copy-example:
+    # Copies the contents of the test directory.
+    # To access the file, it would be found at ./file
+    COPY test .
+
+    # Also copies the contents of the test directory.
+    # To access the file, it would be found at ./file
+    COPY test/* .
+
+    # Copies the whole test folder.
+    # To access the file, it would be found at ./test/file
+    COPY --dir test .
```

Comment:
```suggestion
# Copies the contents of the test directory.
# To access the file, it would be found at ./file
COPY test .

# Also copies the contents of the test directory.
# To access the file, it would be found at ./file
COPY test/* .

# Copies the whole test folder.
# To access the file, it would be found at ./test/file
COPY --dir test .
```